### PR TITLE
EPMRPP-68917, 68919, 68921, 68923, 68924, 68916, 68903: fixed bugs

### PR DESCRIPTION
**Pull request by oranje322**

### CODE REVIEW

File: app/src/pages/inside/stepPage/modals/makeDecisionModal/elements/itemHeader/itemHeader.scss [hunk 1]
```diff
@@ -65,6 +65,7 @@
   text-decoration: none;
   white-space: nowrap;
   overflow: hidden;
+  max-width: 90%;
```
Comment (chivekrodis) [inline app/src/pages/inside/stepPage/modals/makeDecisionModal/elements/itemHeader/itemHeader.scss:68]: Please recheck this solution for screen 768px and more. 
Also does it expected behavior for screen size 768-990px
image

File: app/src/pages/inside/stepPage/modals/makeDecisionModal/selectDefectManually/selectDefectManually.scss [hunk 2]
```diff
@@ -16,14 +16,17 @@
 
 .ignore-analysis {
   position: absolute;
-  top: -32px;
+  display: flex;
+  top: 105px;
```
Comment (chivekrodis) [inline app/src/pages/inside/stepPage/modals/makeDecisionModal/selectDefectManually/selectDefectManually.scss:20]: Please align checkbox according markup(by text and right side)

File: app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionTabs/makeDecisionTabs.jsx [hunk 3]
```diff
@@ -101,11 +103,22 @@ export const MakeDecisionTabs = ({
...
+                  </p>
+                  <p className={cx('link-wrapper')}>
+                    {formatMessage(messages.pleaseCheck)}
+                    <a
+                      href={'https://reportportal.io/docs/Deploy-Elastic-Search'}
+                      target="_blank"
+                      className={cx('suggestion-link')}
+                    >
+                      <span>{formatMessage(messages.analyzerUnavailableLink)}</span>
+                      <div className={cx('icon')}>{Parser(ExternalLinkIcon)}</div>
```
Comment (chivekrodis) [inline app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionTabs/makeDecisionTabs.jsx:135]: Please do not put div element into p element is not valid layout

File: app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionTabs/makeDecisionTabs.scss [hunk 4]
```diff
@@ -187,6 +226,14 @@
   }
 }
 
+.padding-right-10 {
+  padding-right: 20px;
+}
```
Comment (chivekrodis) [inline app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionTabs/makeDecisionTabs.scss:231]: In my opinion is incorrect class name for padding equal 20px

**Review by chivekrodis — CHANGES_REQUESTED** [review]

**Comment by chivekrodis** [comment]
image
I guess animation should be removed not only if user click on first suggestion item and another too

**Comment by chivekrodis** [comment]
Please align text by center for mid screen size
image

**Comment by AmsterGet** [comment]
> image I guess animation should be removed not only if user click on first suggestion item and another too

No, we discussed with design team and the animation should be stopped only by click on the animated item.

**Review by chivekrodis — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**